docs/index.rst

```diff
@@ -27,6 +27,7 @@ Features
 * :ref:`Build an OCI format image from a Containerfile <build-Containerfile-workflow>` and make it available from the Pulp Registry
 * Host content either `locally or on S3 <https://docs.pulpproject.org/installation/storage.html>`_
 * De-duplication of all saved content
+* Support disconnected and air-gaped environments with pulp import/export facility
```

Comment:
@mdellweg  can you please update this line so it captures that import/export would work only for synced repos?And the changelog entry as well. We will add separately support for pushed repos, once we complete this change in core https://pulp.plan.io/issues/7297
My last suggestion would be to add a note that it is in the tech preview https://docs.pulpproject.org/pulp_container/tech-preview.html